spec/request/address_request_spec.rb

```diff
@@ -166,18 +167,5 @@
         end
       end
     end
-
-    context 'with a 401 malformed token response', vcr: { cassette_name: 'evss/reference_data/401_malformed' } do
```

Comment:
how come this test/cassette was removed?

Reply:
There used to be header/JWT code that was unique to this controller, now it uses the same header that the rest of the EVSS calls use.